packages/osd-plugin-generator/src/integration_tests/generate_plugin.test.ts

```diff
@@ -73,6 +73,7 @@ it('generates a plugin', async () => {
 
   expect(paths.sort((a, b) => a.localeCompare(b))).toMatchInlineSnapshot(`
     Array [
+      <absolute path>/plugins/.DS_Store,
```

Comment:
is this necessary?

Reply:
Changes look good, this just seems specific to macs and unclear why this needs to change